ui/src/main/java/com/midtrans/sdk/ui/views/banktransfer/list/BankTransferListActivity.java

```diff
@@ -0,0 +1,101 @@
+package com.midtrans.sdk.ui.views.banktransfer.list;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.Toolbar;
+
+import com.midtrans.sdk.ui.R;
+import com.midtrans.sdk.ui.abtracts.BaseActivity;
+import com.midtrans.sdk.ui.adapters.BankTransferListAdapter;
+import com.midtrans.sdk.ui.constants.Constants;
+import com.midtrans.sdk.ui.models.PaymentMethodModel;
+import com.midtrans.sdk.ui.utils.PaymentMethodUtils;
+import com.midtrans.sdk.ui.views.banktransfer.payment.BankTransferPaymentActivity;
+import com.midtrans.sdk.ui.widgets.DefaultTextView;
+
+import java.util.List;
+
+/**
+ * Created by ziahaqi on 3/30/17.
+ */
+
+public class BankTransferListActivity extends BaseActivity implements BankTransferListAdapter.BankTransferListener {
+    public static final String ARGS_BANK_LIST = "args.banks";
+
+    private RecyclerView rvBankList;
+    private DefaultTextView tvTotalAmount;
+    private DefaultTextView tvTitle;
+
+    private BankTransferListAdapter bankTransferListAdapter;
+    private BankTransferListPresenter presenter;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_banktransfer_list);
+        initProperties();
+        bindView();
+        setupView();
+        initThemeColor();
+    }
+
+    private void initProperties() {
+        presenter = new BankTransferListPresenter();
+        bankTransferListAdapter = new BankTransferListAdapter(this);
+        List<String> bankList = getIntent().getStringArrayListExtra(ARGS_BANK_LIST);
+        List<PaymentMethodModel> bankPaymentMethods = PaymentMethodUtils.createBankPaymentMethods(this, bankList);
+        bankTransferListAdapter.setData(bankPaymentMethods);
+    }
+
+    private void setupView() {
+        setSupportActionBar(toolbar);
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
```

Comment:
i have override `setSupportActionBar(toolbar)` method in the parent class, so it will use custom drawable icon automatically if we call that method.